nf_core/__main__.py

```diff
@@ -594,7 +594,8 @@ def create_test_yml(ctx, tool, run_tests, output, force, no_prompts):
 @click.option("-a", "--all", is_flag=True, help="Run on all modules")
 @click.option("--local", is_flag=True, help="Run additional lint tests for local modules")
 @click.option("--passed", is_flag=True, help="Show passed tests")
-def lint(ctx, tool, dir, key, all, local, passed):
+@click.option("--fix", is_flag=True, help="Fix the module version if a newer version is available")
```

Comment:
Here I'm not sure if `--fix` alone is a good flag name or it would be better to call it `--fix_version`. When running the `nf-core modules lint` with a `--fix` option, I would expect that this fixes potentially all the linting errors and not only the tool version. What do you think?

Reply:
Or maybe in a similar way that `nf-core lint --fix` works, by providing which kind of linting failure should be fixed. E.g. `nf-core lint --fix versions`. Not sure if it is possible to implement like this in this case

Reply:
Yes, that's true, it can also be misleading with the `--fix` option from the pipeline linting. I will do some refactoring to `--fix_version`

Reply:
I've checked the functionality of the `--fix` flag in the pipeline linting and I think it's different from what we have here with the version, as it allows the different lint tests, but the version fix is included in the linting of main.nf.
I changed the name of the flag to `--fix-version`, and we can implement the option to run specific lint tests in a different PR, do you agree?